src/components/ChoiceGrid/items.js

```diff
@@ -196,6 +198,31 @@ const ChoiceGridItems = () => {
             </div>
           </Link>
         </li>
+        <li className={`${Styles.choiceGridItem} ${Styles.choiceEat}`}>
```

Comment:
This may also be incorrect? Lots of eat mentions.

Reply:
I need the choicePower style and the powerIcon to adjust the logic from the Eat logic in the choicegrid component